Source/WebKit/UIProcess/Launcher/cocoa/ExtensionProcess.mm

```diff
@@ -69,8 +75,41 @@
 {
     NSError *error = nil;
     RetainPtr<BEProcessCapabilityGrant> grant;
-    WTF::switchOn(m_process, [&] (auto& process) {
+    WTF::switchOn(m_process, [&] (RetainPtr<BEWebContentProcess> process) {
+        grant = [process grantCapability:capability error:&error];
+    }, [&] (RetainPtr<BENetworkingProcess> process) {
+        grant = [process grantCapability:capability error:&error];
+    }, [&] (RetainPtr<BERenderingProcess> process) {
         grant = [process grantCapability:capability error:&error];
+    }, [&] (RetainPtr<_SEExtensionProcess> process) {
+    });
+    return grant;
+}
+
+PlatformGrant ExtensionProcess::grantCapability(PlatformCapability capability) const
+{
+    NSError *error = nil;
+    PlatformGrant grant;
+    WTF::switchOn(m_process, [&] (RetainPtr<BEWebContentProcess> process) {
```

Comment:
Same thing here, can't this use am `auto&` lambda parameter? Seems like most lambdas are identical